src/Sylius/Behat/Context/Ui/Admin/MoveToEditPageFromProductShowPageContext.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Behat\Context\Ui\Admin;
+
+
+use Behat\Behat\Context\Context;
+use Sylius\Behat\Page\Admin\Product\UpdateSimpleProductPageInterface;
+use Sylius\Behat\Page\Admin\ProductVariant\UpdatePage;
+use Sylius\Component\Core\Model\ProductInterface;
+use Sylius\Component\Core\Model\ProductVariantInterface;
+use Webmozart\Assert\Assert;
+
+final class MoveToEditPageFromProductShowPageContext implements Context
```

Comment:
Name of this context is not consistent with others. However, is this context necessary? These steps could be in `ManagingProductsContext` and `ManagingProductVariantsContext` contexts

Reply:
I created new context because `ManagingProductsContext` and `ManagingProductVariantsContext` contain the same functions name what couses conficts.